### ISSUE TITLE
libretro-db should use Name from dat, not from metadat

### ISSUE THREAD

**Issue by sergiobenrocha2** [libretro/RetroArch#2935]
libretro-db uses names from the metadats instead from the dats. We update the dat, names are updated but we forget the metadats, and the libretro-db just pick the names from the last.

For example, in the genesis dat we have:

```
F-117 Stealth - Operation - Night Storm (Japan)
```

but in the metadats we have:

```
F-117 Stealth - Operation Night Storm (Japan)
```

It's been a problem to match with thumbnails, since the names there are the last from no-intro (well, we are updating them).

thanks Juan to get this.

**Comment by Kivutar** [comment]
What about removing all the names from the metadat ?

**Comment by twinaphex** [comment]
No, this is by design. 

This is so that we can override names.

**Comment by sergiobenrocha2** [comment]
Updating the names in the metadats is quite unmaintainable...

**Comment by sergiobenrocha2** [comment]
Need to build the rdb with the dat in end, otherwise it'll get the names from metadat

Wrong:
```
./c_converter ../../../libretro-database/Sega\ -\ Mega\ Drive\ -\ Genesis.rdb rom.crc ../../../libretro-database/libretro-database-sergio/dat/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/developer/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/enhancement_hw/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/publisher/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/releasemonth/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/releaseyear/Sega\ -\ Mega\ Drive\ -\ Genesis.dat
```

Right:

```
./c_converter ../../../libretro-database/Sega\ -\ Mega\ Drive\ -\ Genesis.rdb rom.crc ../../../libretro-database/libretro-database-sergio/metadat/developer/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/enhancement_hw/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/publisher/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/releasemonth/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/metadat/releaseyear/Sega\ -\ Mega\ Drive\ -\ Genesis.dat ../../../libretro-database/libretro-database-sergio/dat/Sega\ -\ Mega\ Drive\ -\ Genesis.dat
```

**Comment by Kivutar** [comment]
> The title metadats are there so that we can override names.

Why would we need to override the names? It can break things like thumbnails.

**Comment by sergiobenrocha2** [comment]
Partially fixed with libretro/libretro-super#324
